adafruit_sht31.py

```diff
@@ -108,36 +108,34 @@ def temperature_and_relative_humidity(self):
     @property
     def temperature(self):
         """The measured relative humidity in percent."""
-        raw_temperature, raw_humidity = self._data()
-        return (-45 + (175 * (raw_temperature/65535)))
+        temperature, relative_humidity = self.temperature_and_relative_humidity
```

Comment:
this can read 
```
    @property
    def temperature(self):
        """The measured relative humidity in percent."""
        return self.temperature_and_relative_humidity[0]
```